plans/.sprint36.md

```
@@ -0,0 +1,1089 @@
+// MediaService
+Media updateMetadata(Long id, Long folderId, Boolean isPublic, List<String> tags)
+Page<Media> findAll(Pageable pageable)
+Media moveToFolder(Long mediaId, Long folderId)
+
+// MediaContainerService
+Optional<MediaContainer> findByMasterMediaId(Long masterMediaId) // alias
+```
+
+### 14.6 Code Review Improvements (Refactoring)
+
+**1. Performance Optimization (P0)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)